travis/test_feats_unit.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+from ..server.characters.feats import Feat
+
+
+def test_feat_class():
+    expected_tuple = ("4", "test feat", "[]", "This is a test feat", "Travis Test", \
+        "timestamp", "True", "0")
+    myFeat = Feat(expected_tuple)
+    assert myFeat.name == "test feat"
+    assert myFeat.nanite_cost == 0
+    assert myFeat.pk_id == 4
```

Comment:
While I recognize that it's a bit banal, truly rigorous testing here would have us asserting on all 8 attributes. And probably another feat with as different-looking attributes as possible. But I'd settle for the former.

Reply:
Not banal; absolutely correct. That's what chapter 8 was about. I was supposed to write the tests at the same time as the feats... Also, we just need WAAAY more tests.